setup.cfg

```diff
@@ -69,6 +69,9 @@ ignore_missing_imports = True
 [mypy-pylogics.*]
 ignore_missing_imports = True
 
+[mypy-FDgrounder.*]
+ignore_missing_imports = True
+
```

Comment:
As before?

Reply:
it depends on https://github.com/whitemech/Plan4Past/pull/264#discussion_r1342008947